starters/deno-oak-denodb/src/main_test.ts

```diff
@@ -0,0 +1,5 @@
+import { assertEquals, testing } from '../dev_deps.ts';
+
+Deno.test(function addTest() {
```

Comment:
Do we need this test or can we hold off with dummy tests until we have something real to test? 🤔

Reply:
I was just suggesting to remove it as it doesn't provide any value, but if you think we should keep it, let's keep it.

Reply:
I can remove it then.

Reply:
Done